cumulus_etl/cli_utils.py

```diff
@@ -17,6 +17,7 @@ def add_auth(parser: argparse.ArgumentParser, *, use_fhir_url: bool = True):
     group = parser.add_argument_group("authentication")
     group.add_argument("--smart-client-id", metavar="ID", help="client ID for SMART authentication")
     group.add_argument("--smart-jwks", metavar="PATH", help="JWKS file for SMART authentication")
+    group.add_argument("--smart-pem", metavar="PATH", help="PEM file for SMART authentication")
```

Comment:
Option B: deprecate `--smart-jwks` (but keep it around secretly) and replace it with `--smart-key`, and figure out whether it's a PEM or a JWKS in realtime -- maybe content sniffing, maybe as lazy as file suffixes, but I don't think PEM suffixes are fully normalized.

I went with the extremely explicit route, but maybe that's worse UX - I'm not sure.

Reply:
I do like hiding the complexity of 'which key format am i using' from the CLI - up to you if you wanna try it or not.

Reply:
I can work on that :+1:

Reply:
Done!